app/v1/controllers/requestParsers/validators/validations/TypeOfLossValidation.scala

```diff
@@ -24,4 +24,9 @@ object TypeOfLossValidation {
   def validate(typeOfLoss: String): List[MtdError] = {
     if (TypeOfLoss.parser.isDefinedAt(typeOfLoss)) NoValidationErrors else List(TypeOfLossFormatError)
   }
+
+  def validateLossClaim(typeOfLoss: String): List[MtdError] = {
```

Comment:
We should re-use this in `CreateLossClaimValidator` by changing its `typeOfLossValidator` method to:

```
  private def typeOfLossValidator: CreateLossClaimRawData => List[List[MtdError]] = { data =>
    List(
      JsonValidation.validate[String](data.body.json \ "typeOfLoss")(TypeOfLossValidation.validateLossClaim)
    )
  }
```